_tips/developing.id.md

```diff
@@ -0,0 +1,638 @@
+  <p>Dikembangkan oleh Kelompok Kerja Edukasi dan Pendampingan (<a href="https://www.w3.org/WAI/EO/">EOWG</a>). Dikembangkan sebagai bagian dari  <a href="https://www.w3.org/WAI/DEV/">proyek WAI-DEV</a>, didanai bersama oleh Komisi Eropa.</p>
+---
+
+{::nomarkdown}
+{% include box.html type="start" h="2" title="Ringkasan" class="full" %}
+{:/}
+
+Halaman ini memperkenalkan beberapa pertimbangan mendasar untuk membantu Anda mulai mengembangkan konten web yang lebih aksesibel bagi penyandang disabilitas. Kiat-kiat ini merupakan praktik yang baik dalam membantu Anda memenuhi persyaratan Pedoman Aksesibilitas Konten Web (WCAG). Ikuti tautan ke persyaratan WCAG terkait, latar belakang terperinci dalam dokumen "Memahami", panduan dari Tutorial, kisah pengguna, dan banyak lagi.
+
+{::nomarkdown}
+{% include box.html type="end" %}
+{:/}
+
+{::options toc_levels="2" /}
+
+{::nomarkdown}
+{% include_cached toc.html type="start" title="Daftar Isi" class="full" %}
+{:/}
+
+-   TOC is created automatically.
+{:toc}
+
+{::nomarkdown}
+{% include_cached toc.html type="end" %}
+{:/}
+
+
+
+## Kaitkan label dengan setiap kontrol pada isian
+
+Gunakan atribut `for` pada elemen `<label>` yang dihubungkan ke atribut `id` dari elemen isiannya, atau gunakan atribut <abbr title="Accessible Rich Internet Applications">WAI-ARIA</abbr>. Dalam situasi tertentu, menyembunyikan elemen `<label>` secara visual mungkin bisa diterima, namun pada sebagian besar kasus, label diperlukan untuk membantu semua pembaca mengerti input apa yang diperlukan.
+
+{::nomarkdown}
+{% include_cached box.html type="start" title="Contoh: Menggunakan atribut <code>for</code> dan <code>id</code>" class="example" %}
+
+<div class="two-column">
+<figure>
+<figcaption>{% include_cached icon.html name="desktop" %} Hasil penyajian</figcaption>
+<label for="username">Nama pengguna</label>
+<input id="username" type="text" name="username">
+</figure>
+<figure>
+<figcaption>{% include_cached icon.html name="code" %} Penggalan Kode</figcaption>
+{:/}
+
+~~~html
+<label for="username">Nama pengguna</label>
+<input id="username" type="text" name="username">
+~~~
+
+{::nomarkdown}
+</figure>
+</div>
+{% include_cached box.html type="end" %}
+{:/}
+
+{::nomarkdown}
+{% include_cached box.html type="start" title="Informasi Lebih Lanjut" class="simple" %}
+{:/}
+
+* **WCAG**
+  * [Label atau Instruksi 3.3.2](/WAI/WCAG21/quickref/#labels-or-instructions) ([Memahami 3.3.2](/WAI/WCAG21/Understanding/labels-or-instructions))
+* **Tutorial**
+  * [Melabeli Kontrol](/tutorials/forms/labels/)
+
+{::nomarkdown}
+{% include_cached box.html type="end" %}
+{:/}
+
+## Sediakan alternatif teks untuk gambar
+
+Pastikan alternatif teks untuk gambar ditambahkan ke semua gambar yang bersifat informatif dan fungsional. Gunakan teks alternatif kosong, `alt=""` untuk gambar dekoratif, atau sertakan gambar tersebut pada CSS. Alternatif teks biasanya disediakan oleh mereka yang bertanggung jawab untuk menulis konten.
```

Comment:
Original content: "Ensure that alternative text for images is added to all informational and functional images."

Translation: "Pastikan alternatif teks untuk gambar ditambahkan ke semua gambar yang bersifat informatif dan fungsional."

@frirasyidi I just want to make sure that the translation does not imply that the images have to be functional AND informative for the alternative text to be necessary. Said differently, we want alt text added to all informative images, and all functional images.
Would you say the current translation could be misleading, or not?

Reply:
@remibetin got it! I changed it to "Pastikan alternatif teks untuk gambar ditambahkan ke semua gambar yang bersifat informatif mau pun fungsional." in the next commit.

"mau pun" is similar to and/or.